pypeclub/OpenPype
# Settings error dialog on show

## Issue
- error dialogs are shown before settings UI is visible and as they are dialogs they are not visibe in Tray so it's easy to loose them

## Changes
- dialogs are shown with small delay

### How to test
1. make a json parse error in any used schema
2. open settings UI
3. error dialog should appear as modal window of settings UI (settings UI should be visible in background and in tasks)
![image](https://user-images.githubusercontent.com/43494761/126505291-2a52a02a-7f5d-45ca-91b9-8103a4008c71.png)


Closes: https://github.com/pypeclub/OpenPype/issues/1796
